### PR TITLE
modify container for a prod installation

### PR DESCRIPTION
**Pull request by pkruk**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,13 +38,14 @@ also you can install VmShepherd manually using a make:
 
   make install
 
-Also we provide a dockerfile which can be used for a development installation:
+There is also a third way with using a docker container:
```
Comment (evemorgen) [inline README.rst]: I'm not so sure that we want to introduce THIRD way to run this. Even 2 may be to much imho

File: README.rst [hunk 2]
```diff
@@ -93,29 +94,20 @@ Firstly build a image:
 
  docker build -t vmshepherd . --rm
 
-Our Dockerfile create a ``ENTRYPOINT`` for our makefile, so basically you can execute make commands in docker container like run or show-docs.
-
-Examples of usage:
-
-* Running vmshepherd application:
+* Running vmshepherd application with config and mounted source code for a development
```
Comment (evemorgen) [inline README.rst]: What does "mounted source code" means? I have no clue, seriously

File: README.rst [hunk 3]
```diff
@@ -38,13 +38,14 @@ also you can install VmShepherd manually using a make:
...
   make install
 
-Also we provide a dockerfile which can be used for a development installation:
+There is also a third way with using a docker container:
 
 ::
 
   docker build -t vmshepherd .
-  docker run -it  -p 8888:8888 -p 8000:8000 vmshepherd run
+  docker run -d --name vmshepherd -v /etc/vmshepherd/confs/:/etc/vmshepherd -p 8888:8888 vmshepherd -c /etc/vmshepherd/settings.yaml
```
Comment (evemorgen) [inline README.rst]: what about port `8000`?

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.6-rc-slim
 
-RUN apt-get -y update; apt-get -qq -y install make sudo
+RUN apt-get -y update; apt-get -qq -y install make sudo graphviz libgraphviz-dev pkg-config
```
[the remark this answers is not in the document]
Comment (evemorgen) [reply Dockerfile]: i bet they are required by docs, not by app itself

File: README.rst (hunk 3 above)
Comment (pkruk) [reply README.rst]: is for a sphinx, well i removed that, and leave pure installation of vmshepherd

**Pull request closed without merging**